### ISSUE TITLE
Case statement with the word 'in' inside a clause breaks highlighting

### ISSUE THREAD

**Issue by friendly-bits** [jeff-hykin/better-shell-syntax#81]
## The code with a problem is:
```
case a in b) echo "the word 'in' triggers the issue."; esac
echo "Now highlighting is wrong"
```

## It looks like:
![image](https://github.com/jeff-hykin/better-shell-syntax/assets/134004289/b1e667a7-92ae-49ef-8653-2015bd1034f8)

**Comment by friendly-bits** [comment]
P.s.
![image](https://github.com/jeff-hykin/better-shell-syntax/assets/134004289/f37a6156-1de5-4784-b163-0cbb537b7057)

**Comment by jeff-hykin** [comment]
Thanks! should be fixed in v1.8.1
